lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx

```diff
@@ -92,29 +96,28 @@ class HelpPanelItem extends React.Component {
               semiBold
               uppercase
               css={`
+                padding-right: 25px;
+                padding-left: 5px;
                 display: table-cell;
                 vertical-align: middle;
                 font-size: 16px;
                 line-height: 17px;
+                color: #575757;
```

Comment:
Another hardcoded, never seen before color (I know it's from the design and you are just following that, but we need to raise it back to Vic when these things come up - in the absence of a palette of defined colours in the designs, we need to draw from existing colors/color-variable-names, and if there are new ones that should be added, we update current variables in a particular app to use those).